Sources/DolbyIORTSCore/StreamOrchestrator.swift

```diff
@@ -115,9 +117,20 @@ open class StreamOrchestrator {
                     }
                 }
                 
+                // Unproject audio whose source has been stopped from the publisher
+                if let lastSource = self.lastSelectedAudioSource,
+                   lastSource.sourceId != source.sourceId,
+                   sources.first(where: { $0.id == lastSource.id }) == nil {
+                    group.addTask {
+                        await self.stateMachine.setPlayingAudio(false, for: lastSource)
```

Comment:
If the source is removed the 'inActive' method will be called and the State machine won't have the source any more and this operation would be invalid ?

Reply:
What are you suggesting here?
1. Remove this setPlayeruAudio() or
2 Move the unproject() the audio into the inActive() method